charts/otel-cloud-stack/values.yaml

```diff
@@ -38,12 +38,11 @@ autoinstrumentation:
   resource: {}
 
 daemonCollector:
+  enabled: true
   name: daemon
-  labels:
-    opentelemetry.io/opamp-managed: "true"
+  opampManaged: false
```

Comment:
I think defaulting to `true` would be best for this and all collectors

Reply:
resolving this thread and discussing here https://github.com/lightstep/otel-collector-charts/pull/76#discussion_r1526465510